src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Appeal.cshtml

```diff
@@ -57,32 +57,117 @@
+                            </fieldset>
+                        </div>
 
                     <div das-highlight-error-for="ApproveModeration" error-class="govuk-form-group--error" class="govuk-form-group">
                         <fieldset class="govuk-fieldset" aria-describedby="ApproveModeration-error">
                             <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                                 <h2 class="govuk-heading-m">Moderation outcome</h2>
                             </legend>
 
-                            <partial name="_ModerationOutcome" for="@Model.ModerationOutcome" />
-                        </fieldset>
-                    </div>
+                                <partial name="_ModerationOutcome" for="@Model.ModerationOutcome" />
+
+                                @if (!Model.HasFinalOutcome)
+                                {
+                                    <span das-validation-for="ApproveModeration" class="govuk-error-message"></span>
+
+                                    <div class="govuk-radios govuk-radios--inline govuk-!-margin-bottom-6">
+                                        <div class="govuk-radios__item">
+                                            <input class="govuk-radios__input"
+                                                   type="radio"
+                                                   value="@ApprovalStatus.Approve"
+                                                   asp-for="ApproveModeration" />
+                                            <label class="govuk-label govuk-radios__label" for="ApproveModeration">
+                                                Approve this outcome
+                                            </label>
+                                        </div>
+                                        <div class="govuk-radios__item">
+                                            <input class="govuk-radios__input"
+                                                   type="radio"
+                                                   id="OptionApproveModerationNo"
+                                                   value="@ApprovalStatus.Overturn"
+                                                   asp-for="ApproveModeration"
+                                                   data-module="das-show-hide-controls"
+                                                   data-aria-controls="conditional-UnsuccessfulExternalText" />
+                                            <label class="govuk-label govuk-radios__label" for="OptionApproveModerationNo">
+                                                Overturn this outcome
+                                            </label>
+                                        </div>
+                                    </div>
+                                }
+                            </fieldset>
+                        </div>
+                    }
+                </div>
+            </div>
+        }
```

Comment:
This is being highlighted as a change. Aren't we supposed to be on a readonly view?